src/codegen/clean-generated.ts

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+import glob from 'glob';
+import path from 'path';
+import rimraf from 'rimraf';
```

Comment:
`rimraf` is not in `package.json`? It used to be there, but was removed in #1944. Not really sure we need it though, as `fs.unlink` and `fs.rmdir` works just fine for this use-case, IMO.